README.md

```diff
@@ -194,6 +194,26 @@ end
   `handle_data` will be invoked as each part arrives.
   An application should never assume how a body will be broken into data parts.*
 
+#### Request/Response flow
+
+It is worth noting what guarantees are given on the request parts passed to the
+Server's `handle_*` functions. It depends on the Server type,
+`Raxx.Server` vs `Raxx.SimpleServer`:
+
+<!-- NOTE: diagram svg files contain the source diagram and can be edited using draw.io -->
+![request flow](assets/request_flow.svg)
+
+So, for example, after a `%Raxx.Request{body: false}` is passed to a Server's `c:Raxx.Server.handle_head/2`
```

Comment:
What's the `c` for?

Reply:
It's a reference to a callback, and `t:` would be a reference to a type. `ex_doc` will render them nicely, without the prefixes, and link to the right thing.

https://hexdocs.pm/elixir/master/writing-documentation.html#recommendations